crates/ra_assists/Cargo.toml

```diff
@@ -11,6 +11,7 @@ doctest = false
 format-buf = "1.0.0"
 join_to_string = "0.1.3"
 rustc-hash = "1.1.0"
+itertools = "0.8.2"
```

Comment:
There's a `0.9.0` version out in the wild, maybe we can use that, just to be up to date here at least.

Reply:
It seems all the other crates in this repo use `0.8.2`, so changing only this crate would mean compiling both versions. IMO it seems nice to use the same version of a dependency across all the crates in a single project like this, just to reduce mental overhead in cases where the API has actually changed. 

I'm not sure how dependency upgrades are typically done in this project, so let me know if there is some existing standard practice, but would you mind if I pushed this to a follow-up PR?